src/lib/Editor/UseCase/Presenter/EditModeSwitch/ViewMode.js

```diff
@@ -10,11 +11,15 @@ export default class ViewMode extends EditMode {
 
     this.#editorHTMLElement = editorHTMLElement
 
-    document.addEventListener('selectionchange', () => {
+    const emitSelectedTextChange = debounce300(() => {
       this.#updateSelectedTextOffsets()
 
       eventEmitter.emit('textae-event.editor.selected-text.change')
     })
+
+    document.addEventListener('selectionchange', () => {
+      emitSelectedTextChange()
+    })
```

Comment:
```suggestion
    document.addEventListener('selectionchange', emitSelectedTextChange)
```

引数に変更がなければ関数オブジェクトをそのまま渡してあげても大丈夫だと思います。

Reply:
イベントリスナーに関数オブジェクトを渡すように修正しました。

[Pass emitSelectedTextChange as a function object to event listener](https://github.com/pubannotation/textae/pull/214/commits/681467c669226764d1af3e8a51c68360f6845611)